src/core/editor.coffee

```diff
@@ -109,6 +109,19 @@ class Editor
       top: bounds.top - containerBounds.top
     }
 
+  scrollCursorIntoView: () ->
+    return unless @selection.range
+    startBounds = this.getBounds(@selection.range.start)
+    endBounds = this.getBounds(@selection.range.end)
```

Comment:
Also a nitpick but can you change this to:

```coffee
endBounds = if (@selection.range.isCursor()) then startBounds else this.getBounds(@selection.range.end)
```

to avoid the extra call if possible (this will likely be the most common case).